hikari/messages.py

```diff
@@ -309,6 +309,18 @@ class MessageReference:
     a guild.
     """
 
+    def make_link(self) -> str:
+        """Generate a jump link to this message or channel in case for follow add messages.
+
+        Returns
+        -------
+        str
+            The jump link to the message.
+        """
+        guild_id_str = "@me" if self.guild_id is None else str(self.guild_id)
+        message_id_str = "" if self.id is None else str(self.id)
```

Comment:
This was copied from the original `make_link`.
I guess this is because mypy/pyright might complain about guild_id_str being a str | hikari.Snowflakeish union?

Reply:
Even then, the `str()` cast in the original is still useless.